contracts/Controller.sol

```diff
@@ -5,6 +5,7 @@ pragma solidity =0.6.10;
 
 pragma experimental ABIEncoderV2;
 
+import {OMath} from "./OMath.sol";
```

Comment:
Can we change the `OMath` naming ? like `Omath` or something... just one uppercase letter in the beginning.